packages/client/components/ActivityLibrary/CreateNewActivity/CreateNewActivity.tsx

```diff
@@ -0,0 +1,263 @@
+import React, {ComponentPropsWithoutRef, useState} from 'react'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
+import graphql from 'babel-plugin-relay/macro'
+import * as RadioGroup from '@radix-ui/react-radio-group'
+import clsx from 'clsx'
+import {Link} from 'react-router-dom'
+
+import newTemplate from '../../../../../static/images/illustrations/newTemplate.png'
+import estimatedEffortTemplate from '../../../../../static/images/illustrations/estimatedEffortTemplate.png'
+
+import {CreateNewActivityQuery} from '~/__generated__/CreateNewActivityQuery.graphql'
+import {ActivityCard} from '../ActivityCard'
+import {ActivityBadge} from '../ActivityBadge'
+
+import IconLabel from '../../IconLabel'
+import {ActivityLibraryHeader} from '../ActivityLibraryHeader'
+import NewMeetingTeamPicker from '../../NewMeetingTeamPicker'
+import sortByTier from '../../../utils/sortByTier'
+import {useHistory} from 'react-router'
+import {AddReflectTemplateMutation$data} from '../../../__generated__/AddReflectTemplateMutation.graphql'
+import useAtmosphere from '../../../hooks/useAtmosphere'
+import useMutationProps from '../../../hooks/useMutationProps'
+import AddReflectTemplateMutation from '../../../mutations/AddReflectTemplateMutation'
+import {Threshold} from '../../../types/constEnums'
+import useRouter from '../../../hooks/useRouter'
+import PrimaryButton from '../../PrimaryButton'
+
+type ActivityType = 'retrospective' | 'poker'
+
+const SUPPORTED_CUSTOM_ACTIVITIES = [
+  {
+    title: 'Process Feedback',
+    type: 'retrospective' as ActivityType,
+    includedCategories: ['retrospective', 'feedback', 'strategy'],
+    image: newTemplate,
+    isEnabled: true,
+    badges: [
+      {category: 'retrospective', theme: 'bg-grape-500 text-white', title: 'Retrospective'},
+      {category: 'feedback', theme: 'bg-jade-400 text-white', title: 'Feedback'},
+      {category: 'strategy', theme: 'bg-rose-500 text-white', title: 'Strategy'}
```

Comment:
-1 Let's reuse `CategoryThemes` and `CATEGORY_ID_TO_NAME` for here

Reply:
good idea!